com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/ConsoleProgress.java

```diff
@@ -78,8 +78,21 @@ private void printProgress() {
     }
 
     @Override
-    public void beginTask(String name, int work) {
-        System.out.print(name);
+    public void beginTask(Task name, int work) {
+        System.out.print(switch (name) {
+            case BUNDLING -> "Bundling...";
+            case BUILDING_ENGINE -> "Building engine...";
+            case CLEANING_ENGINE -> "Cleaning engine";
+            case DOWNLOADING_SYMBOLS -> "Downloading symbols...";
```

Comment:
If we wanted to, we could retain the original message string by injecting `work` into this: `"Downloading %s symbols..."`.

Reply:
Ohhh nice!